fdbserver/VersionedBTree.actor.cpp

```diff
@@ -4767,7 +4767,10 @@ struct BoundaryRefAndPage {
 // DecodeBoundaryVerifier provides simulation-only verification of DeltaTree boundaries between
 // reads and writes by using a static structure to track boundaries used during DeltaTree generation
 // for all writes and updates across cold starts and virtual process restarts.
-struct DecodeBoundaryVerifier {
+class DecodeBoundaryVerifier {
+	Reference<IPageEncryptionKeyProvider> keyProvider;
+	static std::map<std::string, DecodeBoundaryVerifier> verifiers;
```

Comment:
It's not clear to me why this had to be moved, it looks like getVerifier() to get the long-lived verifier for a particular instance is still used so it could store this map internally.

Reply:
I moved this back into the scope of the function.